pyrogram/types/inline_mode/inline_query_result_audio.py

```diff
@@ -0,0 +1,132 @@
+#  it under the terms of the GNU Lesser General Public License as published
+#  by the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  Pyrogram is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with Pyrogram.  If not, see <http://www.gnu.org/licenses/>.
+
+from typing import Union
+
+from pyrogram import raw
+from pyrogram import types
+from pyrogram.parser import Parser
+from pyrogram.types import InlineQueryResult
+
+
+class InlineQueryResultAudio(InlineQueryResult):
+    """Link to an audio file.
+    By default, this audio file will be sent by the user with optional caption.
+    Alternatively, you can use *input_message_content* to send a message with the specified content instead of the
+    audio.
+    Parameters:
+        audio_url (``str``):
+            A valid URL for the audio file.
+        title (``str``):
+            Title for the result.
+        duration (``int``, *optional*):
+            Duration of the audio in second.
+        voice (``bool``, *optional*):
+            Whether this file should be sent as voice message (default=False).
+        performer (``str``, *optional*):
+            Audio artist.
+        mime_type (``str``, *optional*):
+            File mime type (default="audio/ogg").
+        thumb_url (``str``, *optional*):
+            URL of the static thumbnail for the result (jpeg or gif)
+            Defaults to the value passed in *animation_url*.
+        id (``str``, *optional*):
+            Unique identifier for this result, 1-64 bytes.
+            Defaults to a randomly generated UUID4.
+        description (``str``, *optional*):
+            Short description of the result.
+        caption (``str``, *optional*):
+            Caption of the photo to be sent, 0-1024 characters.
+        parse_mode (``str``, *optional*):
+            By default, texts are parsed using both Markdown and HTML styles.
+            You can combine both syntaxes together.
+            Pass "markdown" or "md" to enable Markdown-style parsing only.
+            Pass "html" to enable HTML-style parsing only.
+            Pass None to completely disable style parsing.
+        reply_markup (:obj:`~pyrogram.types.InlineKeyboardMarkup`, *optional*):
+            An InlineKeyboardMarkup object.
+        input_message_content (:obj:`~pyrogram.types.InputMessageContent`):
+            Content of the message to be sent instead of the photo.
+    """
+
+    def __init__(
+            self,
+            audio_url: str,
+            title: str,
+            duration: int = 0,
+            voice: bool = False,
+            performer: str = "",
+            mime_type: str = "audio/ogg",
```

Comment:
There's no mime_type parameter in the Bot API, is this really useful?

Reply:
I thought it would be useful in future, anyways it has default value so it won't cause errors or something.
I will delete mime_type param and commit ASAP